lib/pos_epoch_complete_hook.go

```diff
@@ -118,16 +118,6 @@ func (bav *UtxoView) runEpochCompleteStateTransition(blockHeight uint64, blockTi
 		return nil, errors.Wrapf(err, "runEpochCompleteStateTransition: problem rewarding snapshot stakes: ")
 	}
 
-	// Delete expired nonces
-	prevNonces := bav.GetTransactorNonceEntriesToDeleteAtBlockHeight(blockHeight)
```

Comment:
I'd add a TODO comment here that explains why we're not deleting nonces in this location.